### PR TITLE
AdminVenueView title bar without injected class name

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Sub-PR of 
- https://github.com/sparkletown/sparkle/pull/2607

### CODE REVIEW

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 1]
```diff
@@ -130,61 +130,65 @@ export const AdminVenueView: React.FC = () => {
...
-              Visit Space
+        <div className="AdminVenueView">
+          <AdminTitleBar>
+            <ButtonNG onClick={navigateToHome} iconName={faArrowLeft}>
+              Back to Dashboard
             </ButtonNG>
+            <AdminTitle>Edit {space.name}</AdminTitle>
+            <div>
+              <ButtonNG variant="danger" onClick={showDeleteModal}>
+                Delete space
```
Comment (alexey-petrov) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:141]: let's use `SPACE_TAXON.lower` here

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 2]
```diff
@@ -130,61 +130,65 @@ export const AdminVenueView: React.FC = () => {
...
+              </ButtonNG>
+              <ButtonNG
+                isLink
+                newTab
+                linkTo={spaceSlug ? venueInsideUrl(spaceSlug) : undefined}
+                variant="primary"
+              >
+                Visit Space
```
Comment (alexey-petrov) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:149]: `SPACE_TAXON.capital`

**Review by alexey-petrov — CHANGES_REQUESTED** [review]
A few strings to replace with constants

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: ed859daf2416e43a7a4f6fc045ff2f7441f8dec1

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: ed859daf2416e43a7a4f6fc045ff2f7441f8dec1

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**